src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/FieldTests.vb

```diff
@@ -45,6 +45,25 @@ End Structure
             CompilationUtils.AssertNoDeclarationDiagnostics(compilation)
         End Sub
 
+        <Fact>
+        Public Sub TupleAPIs()
+            Dim compilation = CompilationUtils.CreateCompilationWithMscorlib(
+<compilation name="AAA">
+    <file name="a.vb">
+Public Class C
+    Shared ch1 as C
+End Class
+    </file>
+</compilation>)
+            Dim globalNS = compilation.SourceModule.GlobalNamespace
+            Dim structC = DirectCast(globalNS.GetMembers().Single(), NamedTypeSymbol)
+            Dim field = DirectCast(structC.GetMembers()(1), FieldSymbol)
+            Dim fieldType = DirectCast(field.Type, INamedTypeSymbol)
+
+            Assert.False(fieldType.IsTupleType)
+            Assert.Equal(Nothing, fieldType.TupleElementTypes)
```

Comment:
I have no idea what XUnit does in this case. Consider verifying that the array .IsDefault is true instead.